src/main/java/uk/gov/justice/digital/hmpps/whereabouts/services/AttendanceService.kt

```diff
@@ -53,6 +53,18 @@ class AttendanceService(
     return attendance.map { toAttendanceDto(it) }.toSet()
   }
 
+  fun getAttendanceForBookingsOverDateRange(prisonId: String?,  bookings: Set<Long?>?, fromDate: LocalDate, toDate: LocalDate?,
```

Comment:
Does bookings here have to be nullable? It's required and set nullable at the controller level.  Same for prison.

Reply:
good point, have tightened up